tests/phpunit/payment-methods/class-wc-stripe-upe-payment-method-oc-test.php

```
@@ -20,22 +20,35 @@ public function test_instance() {
 		}
+		try {
+			if ( is_array( $payment_details ) ) {
+				$payment_details = json_decode( wp_json_encode( $payment_details ) );
+			}
+			if ( ! empty( $query_params ) ) {
+				$_GET = array_merge( $_GET, $query_params );
+			}
 
-		$payment_method = new WC_Stripe_UPE_Payment_Method_OC();
-		$actual         = $payment_method->get_title( $payment_details );
+			$payment_method = new WC_Stripe_UPE_Payment_Method_OC();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '43', column '26'). (undefined)

(MissingImport)